### ISSUE TITLE
TypeError: 'NoneType' object is not subscriptable

### ISSUE THREAD

**Issue by holdestmade** [BottlecapDave/HomeAssistant-OctopusEnergy#1066]
### Describe the bug

Getting these in logs, one for gas and one for electric.

Not sure if it's because the end times all show null ?

![Untitled](https://github.com/user-attachments/assets/142c2f75-86d4-447d-b685-cf970e0a8371)

### Reproduction steps

na

### Expected behaviour

na

### Tariff Code

 E-1R-FLUX-IMPORT-23-02-14-M

### Integration Version

13.1.0

### Home Assistant Version

2024.11.0

### Fresh Install?

Not specified

### Home Assistant Logs

This error originated from a custom integration.

Logger: homeassistant
Source: custom_components/octopus_energy/gas/current_accumulative_consumption_kwh.py:93
integration: Octopus Energy (documentation, issues)
First occurred: 10:21:41 am (10 occurrences)
Last logged: 10:25:32 am

Error doing job: Task exception was never retrieved (None)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 266, in _handle_refresh_interval
    await self._async_refresh(log_failures=True, scheduled=True)
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 481, in _async_refresh
    self.async_update_listeners()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 176, in async_update_listeners
    update_callback()
  File "/config/custom_components/octopus_energy/gas/current_accumulative_consumption_kwh.py", line 93, in _handle_coordinator_update
    standing_charge = self._standing_charge_coordinator.data.standing_charge["value_inc_vat"] if self._standing_charge_coordinator is not None and self._standing_charge_coordinator.data is not None else None
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable



This error originated from a custom integration.

Logger: homeassistant
Source: custom_components/octopus_energy/electricity/current_accumulative_consumption.py:113
integration: Octopus Energy (documentation, issues)
First occurred: 10:21:41 am (10 occurrences)
Last logged: 10:25:32 am

Error doing job: Task exception was never retrieved (None)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 266, in _handle_refresh_interval
    await self._async_refresh(log_failures=True, scheduled=True)
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 481, in _async_refresh
    self.async_update_listeners()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 176, in async_update_listeners
    update_callback()
  File "/config/custom_components/octopus_energy/electricity/current_accumulative_consumption.py", line 113, in _handle_coordinator_update
    standing_charge = self._standing_charge_coordinator.data.standing_charge["value_inc_vat"] if self._standing_charge_coordinator is not None and self._standing_charge_coordinator.data is not None else None
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable

### Confirmation

- [X] I confirm that I cannot find my solution within the [documentation](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy)
- [X] I confirm that I cannot find my solution within the [FAQ](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/faq)

**Comment by BottlecapDave** [comment]
Hello and sorry you're seeing this issue. I'll try and get a fix out for this in the next release.

**Comment by BottlecapDave** [comment]
This should now be resolved in v13.1.1

**Comment by holdestmade** [comment]
Thanks I'll give it a try
